entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -2,17 +2,545 @@
+This feature was contributed by [@m4ss1m0g](https://github.com/m4ss1m0g). Many thanks!
+
+### Default max batch size for SQL Server
+
+The default maximum batch size for SaveChanges on SQL Server is now 42. This value was obtained through [analysis of batching performance](https://github.com/dotnet/efcore/issues/9270), and not because we have discovered the Ultimate Question to Live, the Universe, and Everything.
+
+### Default environment to Development
+
+The EF Core command line tools now automatically configure the `DOTNET_ENVIRONMENT` and `ASPNETCORE_ENVIRONMENT` environment variables to "Development". This improves the experience for working with ASP.NET Core during development. See [#19903](https://github.com/dotnet/efcore/issues/19903).
+
+### Better migrations column ordering
+
+The columns for un-mapped base classes are now ordered after other columns for mapped entity types. Note this only impacts newly created tables. The column order for existing tables remains unchanged. See [#11314](https://github.com/dotnet/efcore/issues/11314).
+
+### Query improvements
+
+EF Core 5.0 RC1 contains some additional query translation improvements:
+
+* Translation of `is` on Cosmos--see [#16391](https://github.com/dotnet/efcore/issues/16391)
+* User mapped functions can now be annotated to control null propagation--see [#19609](https://github.com/dotnet/efcore/issues/19609)
+* Support for translation of GroupBy with conditional aggregates--see [#11711](https://github.com/dotnet/efcore/issues/11711)
+* Translation of Distinct operator over group element before aggregate--see [#17376](https://github.com/dotnet/efcore/issues/17376)
+
+### Model building for fields
+
+Finally for RC1, EF Core now allows use of the lambda methods in the ModelBuilder for fields as well as properties. For example, if you are adverse to properties for some reason and decide to use public fields, then these can now be mapped using the lambda builders:
+
+```c#
+public class Post
+{
+    public int Id;
+    public string Name;
+    public string Category;
+    public int BlogId;
+    public Blog Blog;
+}
+
+public class Blog
+{
+    public int Id;
+    public string Name;
+    public ICollection<Post> Posts;
+}
+```
+
+```c#
+protected override void OnModelCreating(ModelBuilder modelBuilder)
+{
+    modelBuilder.Entity<Blog>(b =>
+    {
+        b.Property(e => e.Id);
+        b.Property(e => e.Name);
+    });
+
+    modelBuilder.Entity<Post>(b =>
+    {
+        b.Property(e => e.Id);
+        b.Property(e => e.Name);
+        b.Property(e => e.Category);
+        b.Property(e => e.BlogId);
+        b.HasOne(e => e.Blog).WithMany(e => e.Posts);
+    });
+}
+```
+
+While this is now possible, we are certainly not recommending that you do this. Also, note that this does not add any additional field mapping capabilities to EF Core, it only allows the lambda methods to be used instead of always requiring the string methods. This is rarely useful since fields are rarely public.
```

Comment:
```suggestion
While this is now possible, we are certainly not recommending that you do this. Also, note that this does not add any additional field mapping capabilities to EF Core; it only allows the lambda methods to be used instead of always requiring the string methods. This isn't very useful since fields are rarely public.
```

Reply:
Docs tool recommended not using very as it does not add much value to reader in direct way. If we want to put emphasize then we should put the sentence in a different way.

Reply:
"This is seldom useful since fields are rarely public."